### DISCUSSION TITLE
Value changed in HA will be overwritten again, In HA geänderter Wert wird wieder überschrieben

### DISCUSSION THREAD

**Discussion by UDicke** [marq24/ha-evcc#24] [Q&A] [answered]
wenn ich z.B. über eine Automatisierung den Wert vom Entity number.evcc_battery_grid_charge_limit ändere, ist der kurze Zeit später wieder auf dem Wert der in evcc eingestellt ist. 
Das gleiche passiert wenn ich den Wert der Entität über "https://homeassistant.local:8123/developer-tools/state" ändere,
Woher kommt das und wie kann ich das verhindern?

For example, if I change the value of the entity number.evcc_battery_grid_charge_limit via automation, a short time later it is back to the value set in evcc. 
The same thing happens when I change the value of the entity via "https://homeassistant.local:8123/developer-tools/state"
Where does this come from and how can I prevent it?

**Comment by marq24** [comment, marked as answer]
IMHO you must set the value via an action (and not just set another entity state)...

Setting the value via 'action'

```
action: number.set_value
target:
  entity_id: number.evcc_battery_grid_charge_limit
data:
  value: "0.2"
```

**Reply by UDicke** [reply]
Yes, thank you!
this did not work:
![Screenshot 2024-10-10 064855](https://github.com/user-attachments/assets/79d3c788-39aa-4201-acea-0caa8cc99c1e)

after a small change it works fine;
![Screenshot 2024-10-10 064946](https://github.com/user-attachments/assets/daa1eb45-9869-45ed-b56c-747db6665068)
😊
